packages/web-components/src/components/rux-segmented-button/rux-segmented-button.tsx

```diff
@@ -30,6 +30,13 @@ export class RuxSegmentedButton {
      */
     @Prop({ reflect: true, mutable: true }) selected: string = ''
 
+    /**
+     * Changes size of segmented button from medium to small or large by setting sizing classes
```

Comment:
nit: I would just say changes the size here. the specific classes are just implementation details and hidden by the shadow dom

Reply:
I'm fine with that. I based it off of the rux-button size description. Want me to change that too?